ebos/eclwriter.hh

```diff
@@ -555,6 +641,9 @@ private:
     struct EclWriteTasklet
         : public TaskletInterface
     {
+        Opm::SummaryState& summaryState;
```

Comment:
Apparently this code (which I think is suspect as described before) was already added in #1874, I have no idea why it looks new in this PR?!?

Reply:
> I have no idea why it looks new in this PR?!?

**Probably** a missing rebase.